apps/users/tests/test_services.py

```
@@ -0,0 +1,20 @@
+"""Service layer tests for user helpers."""
+import pytest
+
+from apps.users.models import User
+
+
+@pytest.mark.django_db
+def test_create_user_requires_mobile():
+    """User manager should validate presence of mobile."""
+
+    with pytest.raises(ValueError):
```

Ruff (0.14.0)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)